system/Database/BasePreparedQuery.php

```diff
@@ -147,17 +147,18 @@ abstract public function _execute(array $data): bool;
     abstract public function _getResult();
 
     /**
-     * Explicitly closes the statement.
+     * Explicitly closes the prepared statement.
      */
-    public function close()
+    public function close(): bool
```

Comment:
This is a breaking change. Please add it to the changelog:
https://github.com/codeigniter4/CodeIgniter4/blob/4.3/user_guide_src/source/changelogs/v4.3.0.rst#others-1

Reply:
done